internal/submit/tx_sender.go

```diff
@@ -22,29 +24,34 @@ import (
 	"github.com/cosmos/cosmos-sdk/types/tx/signing"
 	authtxtypes "github.com/cosmos/cosmos-sdk/x/auth/tx"
 	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
+	feemarkettypes "github.com/skip-mev/feemarket/x/feemarket/types"
 
 	"github.com/neutron-org/neutron-query-relayer/internal/config"
 )
 
 const (
 	accountQueryPath             = "/cosmos.auth.v1beta1.Query/Account"
 	simulateQueryPath            = "/cosmos.tx.v1beta1.Service/Simulate"
+	getPricesQueryPath           = "/feemarket.feemarket.v1.Quuery/GasPrice"
```

Comment:
looks like an error, does dynamic prices work at all?
```suggestion
	getPricesQueryPath           = "/feemarket.feemarket.v1.Query/GasPrice"
```